prom2teams/config.ini

```diff
@@ -1,3 +0,0 @@
-[HTTP Server]
```

Comment:
Default .ini still missing. Just the ones in /tests were re-added.

Reply:
this file is not mandatory, default properties are setted in config/settings.py :)